questionnaire/questionnaire.js

```diff
@@ -971,7 +974,7 @@ module.exports = {
                         anyOf: [
                             {
                                 title: 'I have no contact with the offender',
-                                const: 'none'
+                                const: 'i-have-no-contact-with-the-offender'
```

Comment:
Should this be a boolean answer?

Reply:
It's been designed as a check box despite behaving like a yes/no boolean. As such, I've kept it consistent with the rest of the checkboxes. The answer formatter will refactor this as "I have no contact with the offender" which reads a lot better than "None". This page could potentially be redesigned, I have spoken to Julie about it, but for the moment it's staying as is.